ledger_secure_sdk_sys/src/lib.rs

```diff
@@ -61,15 +61,15 @@ unsafe impl critical_section::Impl for CriticalSection {
 /// The heap is stored in the stack, and has a fixed size.
 /// This method is called just before [sample_main].
 #[no_mangle]
-#[cfg(feature = "heap")]
+#[cfg(all(feature = "heap", not(target_os = "nanos")))]
 extern "C" fn heap_init() {
-    const HEAP_SIZE: usize = 8192;
+    const HEAP_SIZE: usize = 1024;
```

Comment:
This change is not part of the PR description, maybe this has been committed by mistake?
I think this should be reverted. With 8K heap by default, there is still plenty of stack available.
Also, this setting may be tunable in the future (see #178)

Reply:
oops yep that's a mistake!